QandA/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from . import views
+
+urlpatterns = [
+    path('',views.index,name='index'),
+]
```

Comment:
Line break at the end as GitHub is also pointing out.

Reply:
Removed the view for now.